src/ansys/fluent/core/field_data_interfaces.py

```diff
@@ -406,6 +407,99 @@ def is_valid(self, name, respect_data_valid=True):
         return name in self(respect_data_valid)
 
 
+class SurfaceData:
+    """
+    A wrapper class for surface data that enables object-style access
+    to nested dictionary structures.
+
+    This class is used to convert nested dictionary-based surface data
```

Comment:
User doesn't know about these internals (conversion from a nested dict-based data). We can say along these lines: Represents the surface data for each surface returned by various methods of the FieldData class. Has these attributes ...

Documenting class attributes - https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

Reply:
Thank you. Done.